modules/auth-allowing-javascript-access-api-server.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * authentication/configuring-corsallowedorigins.adoc
+
+[id="auth-allowing-javascript-access-api-server_{context}"]
+= Allowing JavaScript-based access to the API server from additional hosts
+
+By default, localhost and 127.0.0.1 are whitelisted to access the API server and
```

Comment:
This seems a bit too technical, and it's also not all correct, 127.0.0.1 and localhost are specific for the kube-apiserver configuration, but the oauth-server does not have default CORS set.

You can say that the default configuration allows only the OpenShift console to send requests to the API server.